docs/governance.rst

```diff
@@ -0,0 +1,247 @@
+project to overly rely on the existence of one person. We developed the
+following system with several goals in mind:
+
+- Grow the community.
+- Strive for consensus.
+- Provide a path for when consensus cannot be achieved.
+- Minimize the administrative burden.
+- Maximize the `bus factor`_ of the project.
+- Acknowledge the leadership and time multiple people contribute to our
+  community without demanding more time than any individual can offer.
+  Dividing leadership responsabilies into multiple smaller roles also
+  makes it easier to encourage new people to take on a leadership role
+  without fearing that too much work will be required of them.
+- Openness as a priority:
+
+  - Promote open discussions.
+  - Openness is critical to building trust with the broader community
+  - Openness provides a mechanism for non-leaders to identify and address
+    oversights or mistakes
+  - Openness provides a smoother transition to onboard future leaders
+  - Leadership meetings should be open and notes should be shared unless
+    there are discussions about sensitive personal matters.
+
+This governance structure is a work-in-progress. We welcome both people
+who want to take on a leadership role as well as ideas for our to improve
+this structure.
+
+Leadership
+----------
+
+Contributor
+  A contributor is someone who has made a contribution to tedana. A
+  contribution, can be code, documentation, or conceptual. All contributors are
+  listed in the `all-contributors file`_. The community decides on the content
+  of this file using the same process as any other change to the `Repository`_
+  (see below) allowing the meaning of "Contributor" to evolve independently of
+  the Decision-making rules. Contributors also have the option to be added to
+  the Zenodo file which may be used for authorship credit for tedana.
+    
+
+Maintainer
+  A Contributor is responsible for the long term health of the project and the
+  community. Maintainers have additional authority (see `Decision Making Process`_) helping them to
+  resolve conflicts and increase the pace of the development when necessary.
+  Any maintainer can remove themselves. Any contributor can become a
+  maintainer by request, or by nomination of a current maintainer,  and with the support of the majority of the current
+  maintainers. Current Maintainers:
+
+  +-----------------------------------+
+  | `Logan Dowdle`_ (@dowdlelt)       | 
+  +-----------------------------------+
+  | `Elizabeth DuPre`_ (@emdupre)     |
+  +-----------------------------------+
+  | `Dan Handwerker`_ (@handwerkerd)  |
+  +-----------------------------------+
+  | `Taylor Salo`_ (@tsalo)           |
+  +-----------------------------------+
+  | `Joshua Teves`_ (@jbteves)        |
+  +-----------------------------------+
+  | `Eneko Uruñuela`_ (@eurunuela)    |
+  +-----------------------------------+
+  | `Kirstie Whitaker`_ (@KirstieJane)|
+  +-----------------------------------+
+
+Steering committee
```

Comment:
Maybe you could add a link to the _Steering Committee_ section below.

Reply:
Added in ff7f48b